.github/workflows/cppcheck-monitor.yaml

```diff
@@ -54,7 +52,3 @@ jobs:
         with:
           name: cppcheck-report
           path: cppcheck-report.xml
```

Comment:
```suggestion
          path: cppcheck-report.xml

      - name: Fail the job if Cppcheck failed
        if: steps.cppcheck.outcome == 'failure'
        run: exit 1
```

Reply:
> This action can stay to run daily forever. Maybe some files will go past cppcheck somehow, this will warn us in general.

Thanks for the comment. Agreed that it is better to make it fail on failure given that this CI will stay forever.

Fixed: https://github.com/autowarefoundation/autoware.universe/commit/2ed692bee996ebde3c22238b70844cde823891c5